### PR TITLE
✨ ENHANCE: 2 make README file relevant and useful

### PR DESCRIPTION
**Pull request by Abdullah-dev1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,75 @@
...
+- Component-based-structure
 
-## Learn More
+**Major Dependencies:**
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [email-js](https://www.npmjs.com/package/email-js)
+- [emailjs-com](https://www.npmjs.com/package/emailjs-com)
+- [react-icons](https://www.npmjs.com/package/react-icons)
+- [web-vitals](https://www.npmjs.com/package/web-vitals)
```
Comment (connect-bilal) [inline README.md:52]: Remove Web vitals.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,75 @@
...
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## Structure
 
-### Code Splitting
+```bash
+ /public
+    /index.html
```
Comment (connect-bilal) [inline README.md:58]: Some files don't include a / at the start, and that's inconsistent.

File: README.md [hunk 3]
```diff
@@ -1,70 +1,75 @@
...
-Your app is ready to be deployed!
+**Step 4:** Run the project:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+npm start
+```
 
-### `npm run eject`
+## Technical documentation
```
Comment (connect-bilal) [inline README.md:35]: D should be in Uppercase

File: README.md [hunk 4]
```diff
@@ -1,70 +1,75 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+For a live demo visit, https://muhammad-abdullah-portfolio-62tl7y4i3.vercel.app
```
Comment (connect-bilal) [inline README.md:75]: Showing 404 page

File: README.md [hunk 5]
```diff
@@ -1,70 +1,75 @@
...
+git clone https://github.com/Abdullah-dev1/react-portfoilo.git
+```
 
-### `npm test`
+**Step 2:** Navigate into the project folder:
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```bash
+cd react-portfoilo
```
Comment (connect-bilal) [inline README.md:20]: Check spellings, also check repository name carefully.
portfoilo -> portfolio

**Review by connect-bilal — CHANGES_REQUESTED** [review]
Please review your work thoroughly before submitting the PR.

**Review by connect-bilal — CHANGES_REQUESTED** [review]
Unnecessary changes.

**Comment by Abdullah-dev1** [comment]
Declined

**Pull request closed without merging**